inference-test-tool/run.py

```diff
@@ -148,6 +155,10 @@ def parse_args():
         action='store_true')
     parser.add_argument("-b", "--bounding_box_model", default=False, help="If the model's output are bounding boxes",
         action='store_true')
+    parser.add_argument("-cl", "--classification_model", default=False, help="If the model's output are labels",
```

Comment:
Would be good to also add these two commands to the Readme (section "Testing the inference server")